cloudant-sync-datastore-core/src/main/java/com/cloudant/library/LibraryVersion.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2016 IBM Corp. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ *  except in compliance with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the
+ *   License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  either express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+package com.cloudant.library;
+
+import com.cloudant.sync.util.Misc;
+import com.google.common.io.Resources;
+
+import java.net.URL;
+import java.util.Properties;
+
+/**
+ * A Class to provide the user-agent string to the HTTP layer.
```

Comment:
Be specific to _this_ classes role -- you're describing the interface here. "Provides a HTTP User-Agent string for CloudantSync"